research/conditional/train/configs/baselines/common.yaml

```diff
@@ -0,0 +1,27 @@
+time: "2-00:00:00"
+n_gpus: 1
+interactive_debug: false
+
+# train params
+params:
+  # base bass
+  mixed_precision: true
+
+  # logging etc.
+  use_neptune: true
+  project_name: "pmtest/llm-random"
+  name: "f-bench"
```

Comment:
yeah yeah, f them

Reply:
BTW, should we change names in child configs as well? Basically in each and every config?

Reply:
yes, the name here does not matter too much.